sidebars.js

```diff
@@ -45,13 +45,6 @@ module.exports = {
       "html-css/week-3/homework",
       "html-css/week-3/instructors",
     ],
-    Challenges: [
```

Comment:
This was removed since it's part of the module's homework, and not an additional extra challenge

Reply:
Lets keep this - the idea is that other challenges can be added to that repo in the future.

Reply:
@ChrisOwen101 Ah, I wasn't aware that the repo might be updated in the future. I've added it back in 👍